tests/MailThiefTest.php

```diff
@@ -307,4 +307,20 @@ public function test_get_body_raw()
 
         $this->assertEquals('Raw text content', $mailer->lastMessage()->getBody());
     }
+
+    public function test_undefined_mailthief_message_methods_do_not_cause_error()
+    {
+        $mailer = $this->getMailThief();
+
+        $mailer->send('example-view', [], function ($m) {
+            $m->subject('First message')->addPart('html content', 'text/html');
+        });
+
+        $mailer->send('example-view', [], function ($m) {
+            $m->subject('Second message');
+            $m->addPart('html content', 'text/html');
```

Comment:
Seems valuable to add a test that chains something *after* `addPart` so we can see that the `return $this` is making it chainable as expected, right?